server/server.go

```diff
@@ -1898,8 +1898,11 @@ func (s *Server) LookupAccount(name string) (*Account, error) {
 // This will fetch new claims and if found update the account with new claims.
 // Lock MUST NOT be held upon entry.
 func (s *Server) updateAccount(acc *Account) error {
+	acc.mu.RLock()
+	updated := acc.updated
+	acc.mu.RUnlock()
 	// TODO(dlc) - Make configurable
-	if !acc.incomplete && time.Since(acc.updated) < time.Second {
+	if !acc.incomplete && time.Since(updated) < time.Second {
```

Comment:
We use this approach a lot in code, and the only thing it does, is prevent race detection, not actual race, because we use the `acc.updated` value outside of the mutex.

Reply:
good point, so instead could do the check under the lock

Reply:
exactly.